mlir/lib/Dialect/Rock/Transforms/CleanMath.cpp

```diff
@@ -93,6 +93,10 @@ static LogicalResult deleteTrivialRemainder(DataFlowSolver &solver,
 
 static LogicalResult replaceWithConstant(DataFlowSolver &solver, OpBuilder &b,
                                          OperationFolder &folder, Value value) {
+  IntegerType integerType = llvm::dyn_cast<IntegerType>(value.getType());
+  if (!integerType)
```

Comment:
I add this check because maybeInferredRange->getValue().isUninitialized() now return 0 for a vector, which causes a crash on getIntegerAttr(). I'm not sure if this is entirely correct?